pkg/util/openstack/loadbalancer.go

```diff
@@ -43,9 +44,10 @@ const (
 	OctaviaFeatureTimeout           = 3
 	OctaviaFeatureAvailabilityZones = 4
 
-	loadbalancerActiveInitDelay = 1 * time.Second
-	loadbalancerActiveFactor    = 1.2
-	loadbalancerActiveSteps     = 19
+	waitLoadbalancerInitDelay   = 5 * time.Second
+	waitLoadbalancerFactor      = 1
+	waitLoadbalancerActiveSteps = 36
```

Comment:
this has been increases significantly. any reason?

Reply:
The comment for the previous code is incorrect, we were not using exponential backoff actually, we were using the fixed interval backoff mechanism, so `loadbalancerActiveFactor    = 1.75` doesn't make any sense. I changed this to: Start with 5s delay, sleep 5s (5*1) between each iteration, try 36 times in max, so the total duration is around 5*36=180s, that should be covering most prod and testing environments.

Reply:
> we were not using exponential backoff actually, we were using the fixed interval backoff mechanism,

@lingxiankong Didnt get that, I see we are using "wait.ExponentialBackoff" at L158 , https://pkg.go.dev/k8s.io/apimachinery/pkg/util/wait#ExponentialBackoff , as per my understanding it says it is using longer delays between each retry . Isnt it? Are you suggesting the function is not doing as expected?

Reply:
If you take a look at the implementation of `ExponentialBackoff`, especially `time.Sleep(backoff.Step())`, see how the sleep time is calculated.